components/ModEditSubmissionTopbar.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <div class="flex flex-row justify-between w-full">
+        <div>
+            <button @click="copyId"
+                class="flex flex-row w-90 bg-neutral p-2 m-2 border-2 border-slate-400 rounded hover">
+                ID: {{ submissionId }}
+                <SuccessCheckMark v-if="copySuccess" role="img" title="clipboard copy" class="ml-2 w-6" />
+                <CopyContent v-else role="img" title="successful copy" class="ml-2 w-6" />
+            </button>
+        </div>
+        <div class="flex flex-row justify p-2 font-bold ">
+            <button
+                class="flex justify-center items-center rounded-full bg-white border-slate-400 border-2 w-28 text-sm mr-2 ">{{
+                $t('modEditSubmissionTopNav.saveAndExit') }}</button>
+            <button @click="rejectSubmission"
+                class="flex justify-center items-center rounded-full bg-white border-primary border-2 w-28 text-sm mr-2 ">{{
+                $t('modEditSubmissionTopNav.reject') }}</button>
+            <button @click="acceptSubmission"
+                class="flex justify-center items-center rounded-full bg-primary w-28 text-white text-sm mr-2 ">{{
+                    $t('modEditSubmissionTopNav.approve') }}</button>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import CopyContent from '~/assets/icons/content-copy.svg'
+import SuccessCheckMark from '~/assets/icons/checkmark-square.svg'
+import { Ref, ref } from 'vue';
+
+const props = defineProps({
+    submissionId: {
+        type: String,
+        required: true,
+    }
+}
+)
+
+let copySuccess: Ref<boolean> = ref(false)
```

Comment:
I think we could make this variable name more specific as well. We should know from the name what it is that we're copying. Perhaps something like `copiedSubmissionId`?